rules-java/api/src/main/resources/reports/templates/java_application.ftl

```diff
@@ -101,6 +101,13 @@
     </#if>
 </#macro>
 
+<#macro traverseProject traversal>
+    <@projectModelRenderer traversal.originalProject />
+
+    <#list sortProjectTraversalsByPathAscending(traversal.children) as childTraversal>
+        <@traverseProject childTraversal/>
+    </#list>
+</#macro>
```

Comment:
What does it do when traversing? A little comment with a purpose would be great.